tests/unit/draggable/draggable_core.js

```diff
@@ -318,35 +318,32 @@ asyncTest( "blur behavior", function() {
 	});
 });
 
-test( "ui-draggable-handle assigned to appropriate element", function() {
+test( "ui-draggable-handle assigned to appropriate element", function( assert ) {
 	expect( 5 );
 
 	var p = $( "<p>" ).appendTo( "#qunit-fixture" ),
 		element = $( "<div><p></p></div>" ).appendTo( "#qunit-fixture" ).draggable();
-	ok( element.hasClass( "ui-draggable-handle" ), "handle is element by default" );
-
+	assert.hasClasses( element, "ui-draggable-handle" );
```

Comment:
Why were the blank lines removed in this test? They provide logical groupings of actions and assertions.